en/drafts/originals/text-mining-youtube-comments.md

```diff
@@ -221,19 +220,19 @@ all_data <- all_data |>
+The unique properties of YouTube comments (such as rare words, slang, other languages, or comments that include special characters or emojis), mean that some initial data cleaning is necessary to ensure that each comment contains enough meaningful text data for Wordfish scaling. Comments with little or no usable text data need to be removed, because they will cause substantial skew in a Wordfish model: the model relies on scores given to words with semantic meaning, and comments with fewer than 10 words are unlikely to contain much meaning. This can impact the meaningfulness of results, or may cause the scaling process to fail entirely.
 
-Comments with little or or no usable text data need to be removed, because they will cause substantial skew in a Swordfish model. This can impact the meaningfulness of results, or may cause the scaling process to fail entirely.'
+If you are using an alternative analytical model, you may choose not to remove emojis, links, numbers, mentions, and other miscellaneous detail.
 
-The Wordfish model relies on scores given to words with semantic meaning. Comments with fewer than 10 words are unlikely to contain much meaning. If you are using an alternative analytical model, you may choose not to remove emojis, links, numbers, mentions, and other miscellaneous detail.
+### Remove Stopwords and Punctuation
 
-## Remove Stopwords and Punctuation
 The first pre-processing step is to remove stopwords, which are common words that provide little to no meaningful information about your research question. As [Emil Hvitfeldt and Julia Silge](https://smltar.com/stopwords) explain, whether commonly excluded words would provide meaningful information to your project depends upon your analytical task. For this reason, researchers should think carefully about which words to remove from their dataset.
 
-The words "bronstein", "derrick" and "camry" were notable outliers in our specific dataset; you will likely find that other words are outliers in yours. While these words may reveal some relevance to the political valence of these videos, if you wished to remove these outliers from the visualization, the following code creates a custom stopword list that combines researcher-defined stop words alongside the standard stopword list supplied in the `quanteda` computational text analysis package. To change the custom stopwords, simply replace our words with your own. 
+The words 'bronstein', 'derrick' and 'camry' were notable outliers in our specific dataset; you will likely find that other words are outliers in yours. While these words may reveal some relevance to the political valence of these videos, if you do wish to remove these outliers from the visualization, the following code creates a custom stopword list that combines researcher-defined stopwords alongside the standard stopword list supplied in the `quanteda` computational text analysis package. To change the custom stopwords, simply replace our words with your own. 

```

Comment:
Would you be able to give a little extra clarification as to _why_ these words are considered outliers?